config/api.yml

```diff
@@ -422,8 +422,16 @@
     :options:
     - :collection
     - :subcollection
-    :methods: *70174834086080
+    :methods: *70174834085620
     :klass: ChargebackRateDetail
+    :collection_actions:
+      :post:
+      - :name: create
+    :resource_actions:
+      :post:
+      - :name: edit
```

Comment:
add the "edit" post action here so we support both POST { "action" : "delete", ... } as well as the DELETE methods.  Thanks.